GeosCore/hco_utilities_gc_mod.F90

```diff
@@ -1719,26 +1841,30 @@ SUBROUTINE Get_GC_Restart( Input_Opt, State_Chm, State_Grid, &
       ! levels above the chemistry grid).
       IF ( SpcInfo%Is_InRestart ) THEN
 
-         ! Print the min, max, and sum of each species as it is read from
-         ! the restart file in mol/mol
-         IF ( Input_Opt%amIRoot ) THEN
-            WRITE( 6, 530 ) N, TRIM( SpcInfo%Name ), &
-                            MINVAL( Ptr3D ), MAXVAL( Ptr3D ), SUM ( Ptr3D )
-         ENDIF
-
+         ! Set the mol/mol mixing ratio from the restart file
          !$OMP PARALLEL DO       &
          !$OMP DEFAULT( SHARED  )&
          !$OMP PRIVATE( I, J, L )&
          !$OMP COLLAPSE( 3 )
          DO L = 1, State_Grid%NZ
          DO J = 1, State_Grid%NY
          DO I = 1, State_Grid%NX
-            Spc(N)%Conc(I,J,L) = Ptr3D(I,J,L)
+            IF ( use_HEMCO_for_restart_read ) THEN
+               Spc(N)%Conc(I,J,L) = Ptr3D(I,J,L)
+            ELSE
+               Spc(N)%Conc(I,J,L) = Ptr4D_r8(I,J,L,1)
+            ENDIF
```

Comment:
Now that we have an IF statement in the loop, I think it'd be more efficient if we removed the parallel loop and just used array assignment:
```F90
            IF ( use_HEMCO_for_restart_read ) THEN
               Spc(N)%Conc = Ptr3D
            ELSE
               Spc(N)%Conc = Ptr4D_r8(:,:,:,1)
            ENDIF

Reply:
Good idea. Done!